ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -30,10 +32,90 @@ cat << EOF > ~/ocp-install.yml
   - name: Clone repo
     git:
       repo: https://github.com/karmab/kcli-openshift4-baremetal.git
-      dest: "~/kcli-openshift4-baremetal"
+      dest: ~/kcli-openshift4-baremetal
       version: master
       force: yes
     retries: 5
+  - name: Add master workaround manifest
```

Comment:
@johnbieren 
I think we need to discuss whether we want to split the setup step into multiple steps that make sense

Reply:
Sure, whatever you think is best, you know a lot more about the tests than I do. We will have to make sure we split it in a way that doesn't make us have to reprovision the cluster every time, though (unless you want to reprovision each time)